Assets/Scripts/Managers/PlanetManager.cs

```diff
@@ -117,13 +133,13 @@ public void LeavePlanet()
             }
         }
         ShipMovement.Instance.ResetBoost(); //This resets the ships boost before it goes back to the main scene
-        GameMenuManager.Instance.ReturntoScene("Planet");
+
+        GameMenuManager.Instance.ReturntoScene("Planet Animal Info");
```

Comment:
Make sure this is changed back to Planet in the final merge to staging.

Reply:
Done!